lib/streamTransformers/handlebars.js

```diff
@@ -25,51 +27,162 @@ Handlebars.registerHelper('escapePartial', function(partialName) {
   return partial();
 });
 
-module.exports = function createHandlebarsStream({index, page, partials, debug = false}) {
-  const indexTemplate = fs.readFile(index, 'utf8')
-      .then(compileTemplate);
-  const pageTemplate = fs.readFile(page, 'utf8')
-      .then(compileTemplate);
-  const partialsLoaded = registerGlobPartials(partials, debug);
 
-  let docsBySection;
+function breadcrumbsGetLabel(path, sections) {
+
+  const items = path.split('/');
+  const itemsLength = items.length;
+
+  let label = sections;
+
+  for(let i = 0; i < itemsLength; i++) {
+    if(i < itemsLength - 1 && label[items[i]].hasOwnProperty('children')) {
+      label = label[items[i]]['children'];
+    } else {
+      label = label[items[i]];
+    }
+  }
+
+  return label['title'];
+};
+
+
+function breadcrumbsToArray(path, sections) {
+  let result = [];
+
+  path.split('/')
+      .reduce((finalPath, el) => {
+        const composedPath = finalPath + el;
+        const breadcrumbLabel = breadcrumbsGetLabel(composedPath, sections);
+
+        result.push({
+            label: breadcrumbLabel,
+            path: composedPath + '/'
+          });
+
+        return composedPath + '/';
+      }, '');
+
+  return result;
+}
+
+
+function breadcrumbsTemplate(path, sections) {
+
+  const breadcrumbItems = breadcrumbsToArray(path, sections);
+  const breadcrumbItemsLength = breadcrumbItems.length;
+
+  let html = '';
+
+
+  if(breadcrumbItems.length > 1) {
+    for(let i = 0; i < breadcrumbItemsLength; i++) {
+      if(i == breadcrumbItemsLength - 1) {
+        html += `<span class="breadcrumbs__item breadcrumbs__item--last">${breadcrumbItems[i].label}</span>`;
+      } else {
+        html += `<a href="/${breadcrumbItems[i].path}" class="breadcrumbs__item">${breadcrumbItems[i].label}</a><span class="breadcrumbs__separator"></span>`;
+      }
+    }
+
+    html = `<nav class="breadcrumbs">${html}</nav>`;
+  }
+
+  return html;
+}
 
-  Handlebars.registerHelper('withAllSections', function(options) {
-    return options.fn(docsBySection);
+
+function generateSidebar(sections, index = 0) {
+  let html = '';
```

Comment:
same here, should be a partial. I'd go with a helper that transforms data in to a desired format and then just render the partial using the helper (similar to what `withAllSections` was doing)